### ISSUE TITLE
Provide better examples on how to use slots in group input

### ISSUE THREAD

**Issue by guledali** [wearebraid/vue-formulate#556]
The documentation feels very technical for someone whose not an expert on vue.js

 It's really hard to follow along when it comes to how use slots in group input. I would love to see better examples maybe working examples on how to create your own addMore button or remove button.

**Comment by justin-schroeder** [comment]
Thanks for the feedback. Feel free to submit a PR over on the docs side of things: https://github.com/wearebraid/vueformulate.com
